### PR TITLE
TELCODOCS-1788: OpenShift 4.15 secrets disabled

### PR DESCRIPTION
**Pull request by mikemckiernan**
Authored by @StephenJamesSmith.

Review HTML: 

Special Considerations section: https://nvidia.github.io/cloud-native-docs/review/pr-27/openshift/latest/steps-overview.html#special-considerations-for-openshift-4-15

23.9.2 RN: https://nvidia.github.io/cloud-native-docs/review/pr-27/gpu-operator/latest/release-notes.html#known-limitations

### CODE REVIEW

**Review by mikemckiernan — COMMENTED** [review]
@StephenJamesSmith , I thought that I submitted this feedback before I lost internet connectivity.  PTAL.

**Comment by mikemckiernan** [comment]
Stephen, I put a few notes at the end of Erwan's document. Unless someone knows of a better way, I'm thinking that the `oc get configs.imageregistry.operator.openshift.io cluster -o jsonpath='{.spec.storage}'` command is the best way to predict failure and the need to configure storage for the image registry.

**Comment by StephenJamesSmith** [comment]
@mikemckiernan Saw your sloppy notes. Will you be making these changes to the PR?

File: openshift/steps-overview.rst [hunk 1]
```diff
@@ -41,13 +41,34 @@ You can deploy the Operator on a a newly deployed cluster that was not upgraded
...
+=========================================
 
-   The Driver Toolkit, which enables entitlement-free deployments of the Operator, is available for certain z-streams on OpenShift
-   4.8 and all z-streams on OpenShift 4.9. However, some Driver Toolkit images are broken, so we recommend maintaining entitlements for
-   all OpenShift versions prior to 4.9.9. See :ref:`broken driver toolkit <broken-dtk>` for more information.
+In OpenShift 4.15, secrets are no longer automatically generated when the integrated OpenShift image registry is disabled.
+For more information, refer to the `OpenShift 4.15 Release Notes <https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-auth-generated-secrets>`__.
 
-   You do not need an entitlement on OpenShift Container Platform versions greater than 4.9.9.
+This change affects the instatllation of NVIDIA GPU Operator.
```
Comment (StephenJamesSmith) [inline openshift/steps-overview.rst:51]: s/instatllation/installation

**Review by StephenJamesSmith — COMMENTED** [review]
Found just 1 typo.

**Comment by egallen** [comment]
@mikemckiernan except the typo, the content is good.
/lgtm

File: openshift/steps-overview.rst (hunk 1 above)
Comment (mikemckiernan) [reply openshift/steps-overview.rst:51]: I need to find a different codespell hook.

```
pre-commit run --files openshift/steps-overview.rst 
mixed line ending........................................................Passed
trim trailing whitespace.................................................Passed
check yaml...........................................(no files to check)Skipped
codespell................................................................Passed
```

Thank you for pointing it out to me.

**Pull request merged**